src/textEditor.ts

```diff
@@ -140,6 +141,32 @@ export class TextEditor {
     return vscode.window.activeTextEditor.document.getText(selection);
   }
 
+  /**
+   *  Retrieves the current word at position.
+   *  If current position is whitespace, selects the right-closest word
+   */
+  static getWord(position: Position) : string {
+    const whitespaceRegex = new RegExp("^ *$");
+    let start = position;
+    let end = position.getRight();
+
+    const char = TextEditor.getText(new vscode.Range(start, end));
+    if (whitespaceRegex.test(char)) {
+      start = position.getWordRight();
+    } else {
+      start = position.getWordLeft(true);
```

Comment:
Not necessary for this PR, but I really grow to dislike this bare booleans being passed around. They are very hard to read. Much easier to read would be `position.getWordLeft({ inclusive: true })`